docs/contributing/development.md

```diff
@@ -58,12 +58,13 @@ make create-env
 ```
 
 This will do the following:
-- Start minikube with the profile name "kgw" and enable the Metallb addon. Metallb will expose the envoyfleet services "locally" without needing to port-forward to them.
-- Install our CustomResourceDefinitions
-- Build the docker images, cache them for faster rebuilds and deploy them with Kustomize
-- Deploy an EnvoyFleet.
 
-### Launch Kusk Gateway in your cluster
+- Start minikube with the profile name "kgw" and enable the Metallb add on. Metallb will expose the Envoy Fleet services "locally" without needing to port-forward to them.   
```

Comment:
```suggestion
- Start minikube with the profile name "kusk" and enable the Metallb add on. Metallb will expose the Envoy Fleet services "locally" without needing to port-forward to them.   
```

`kgw` was the old name of the CLI. `kusk` is the new one: we should avoid having reference to `kgw` in the docs